calico/reference/felix/configuration.mdx

```diff
@@ -198,7 +194,13 @@ See the [HOWTO guide](../../operations/ebpf/enabling-ebpf.mdx) for step-by step
 | BPFMapSizeIPSets / <br/> FELIX_BPFMapSizeIPSets                                     | Controls the size of the IPSets map. The IP sets map must be large enough to hold an entry for each endpoint matched by every selector in the source/destination matches in network policy. Selectors such as "all()" can result in large numbers of entries (one entry per endpoint in that case).                                                                               | int                   | 1048576 |
 | BPFMapSizeRoute / <br/> FELIX_BPFMapSizeRoute                                       | Controls the size of the route map. The routes map should be large enough to hold one entry per workload and a handful of entries per host (enough to cover its own IPs and tunnel IPs).                                                                                                                                                                                          | int                   | 262144  |
 | BPFHostConntrackBypass / <br/> FELIX_BPFHostConntrackBypass                         | Controls whether to bypass Linux conntrack in BPF mode for workloads and services.                                                                                                                                                                                                                                                                                                | true,false            | true    |
+| BPFEnforceRPF / <br/>  FELIX_BPFEnforceRPF                                          | Controls RPF on all host interfaces with BPF programs regardless of what is the per-interfaces or global setting. To prevent spoofing, we also enforce it in IPv6 mode. Possible values are `Disabled`, `Strict` or `Loose`.                                                                                                                                                      | string                | "Loose" |
```

Comment:
```suggestion
| BPFEnforceRPF / <br/>  FELIX_BPFEnforceRPF                                          | Controls RPF on all host interfaces with BPF programs regardless of the interface or global settings. To prevent spoofing, we also enforce it in IPv6 mode. Possible values are `Disabled`, `Strict` or `Loose`.                                                                                                                                                      | string                | "Loose" |
```